drivers/clone.c

```diff
@@ -562,6 +571,10 @@ void upsdrv_initinfo(void)
 		battery.runtime.low = strtod(val, NULL);
 	}
 
+	ups.load.off = getval("load.off");
+	ups.load.on = getval("load.on");
+	ups.load.status = getval("load.status");
```

Comment:
Record the user given arguments into the internal state struct, reduces needs for constant `getval`ing them within update loops.